void tomo_db0::IoLoop::run_loop<false, false, false, false, false>()
--- PRE

+++ POST

@@ -601,7 +601,7 @@

 0b2e [ 2] xchg %ax,%ax
 0b30 [ 5] mov $0x1,%esi
 0b35 [ 3] mov %rbx,%rdi
-0b38 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0b38 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0b3d [ 5] jmp <+0x2fb>
 0b42 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b4d [ 3] nopl (%rax)
